pkgs/development/python-modules/openai-whisper/default.nix

```diff
@@ -26,7 +27,6 @@
 buildPythonPackage rec {
   pname = "whisper";
   version = "20231117";
-  pyproject = true;
```

Comment:
Please keep this line, it forces a PEP517 build instead of calling setup.py.

Reply:
Done.